### ISSUE TITLE
Logging

### ISSUE THREAD

**Issue by BryceFury** [BryceFury/tumbly#11]
Logging to be added throughout.

**Comment by simocosimo** [comment]
I think I can help you out with that!
Just a question: do the log files should be rewritten every time the program starts?

**Comment by BryceFury** [comment]
I think the logging should come as part of a try, except on each function so that we can log anything that occurs each time a function is run.

e.g.

def do_something(something):
    try:
        do something()
        logging.info('I did something')
    except:
        logging.critical('Do something broke')

That way as the program improves we can catch any errors, know specifically what they are and where they occur.

If you can think of any other use for logging in the program feel free to suggest it.

**Comment by Algogator** [comment]
"image found at", "image url", "downloaded" in the for loop can be logged, users need to know those details. What do you think?

**Comment by BryceFury** [comment]
@Algogator I agree, I don't they do. The GUI overhall I'm doing has a progress bar instead which I think is a bit better, I will check and merge your PR when I get a chance 👍
